docs/user/adding-new-metrics.md

```diff
@@ -70,7 +70,15 @@ The `lifetime` parameter of a metric defines when it will be reset. There are th
 
 ### Naming things
 
-Choose the category and names of metrics to be as specific as possible.
+Metric identifiers (the combination of a metric's category and name) must be unique across all metrics that are sent by an application.
+This includes not only the metrics defined in the app's `metrics.yaml`, but the `metrics.yaml` of any Glean-using library that that application uses, including Glean itself.
+Therefore, care should be taken to name things specifically enough so as to avoid namespace collisions.
+In practice, this generally involves thinking carefully about the `category` of the metric, more than the `name`.
+
+> **Note:** Duplicate metric identifiers are not currently detected at build time.
+However, the [probe_scraper](https://github.com/mozilla/probe-scraper) process, which runs nightly, will detect this case and e-mail the `notification_emails` associated with the given metrics.
```

Comment:
Do we have a bug for detecting at build time? if so, can we link it here ?

Reply:
No -- I just made a bug and linked to it here.

Reply:
The link in here is still missing.